src/stories/Library/Lists/list-materials/list-materials.scss

```diff
@@ -22,8 +22,9 @@
   flex: 1;
 }
 
```

Comment:
This was a mistake from figma, where it would look like the selected should be styled like the disabled. The spec says otherwise though